### PR TITLE
Deprecate support for CSCC Alpha API

### PR DESCRIPTION
**Pull request by red2k18**

### CODE REVIEW

**Comment by red2k18** [comment]
Fixes #2462

File: google/cloud/forseti/common/gcp_api/securitycenter.py [hunk 1]
```diff
@@ -93,9 +92,7 @@ def __init__(self, **kwargs):
         # pylint: disable=protected-access
         if kwargs.get('gcp_service')._resourceDesc.get('version') == 'v1beta1':
             component = 'organizations.sources.findings'
-        else:
-            # alpha api
-            component = 'organizations.findings'
+
```
Comment (blueandgold) [inline google/cloud/forseti/common/gcp_api/securitycenter.py]: Do you think that we still need the above `if` statement to check if this is a beta api?  I am inclined to think that we can remove it, and rely on this as the default behavior.

File: google/cloud/forseti/common/gcp_api/securitycenter.py [hunk 2]
```diff
@@ -154,23 +151,3 @@ def create_finding(self, finding, organization_id=None, source_id=None,
                 violation_data = (
                     finding.get('source_properties').get('violation_data'))
                 raise api_errors.ApiExecutionError(violation_data, e)
-
```
Comment (blueandgold) [inline google/cloud/forseti/common/gcp_api/securitycenter.py]: ditto: do we still need the `if` check above?  Can we rely on this as the default behavior?  (I think so.)

File: google/cloud/forseti/notifier/notifier.py [hunk 3]
```diff
@@ -173,18 +173,6 @@ def run(inventory_index_id,
                             '%s', source_id)
                         (cscc_notifier.CsccNotifier(inventory_index_id)
                          .run(violations_as_dict, source_id=source_id))
-                    else:
```
Comment (blueandgold) [inline google/cloud/forseti/notifier/notifier.py]: ditto: on the `if` check

File: google/cloud/forseti/notifier/notifiers/cscc_notifier.py [hunk 4]
```diff
@@ -220,31 +191,11 @@ def _send_findings_to_cscc(self, violations, organization_id=None,
                     continue
             return
 
-        # alpha api
-        LOGGER.debug('Sending findings to CSCC with alpha API.')
```
Comment (blueandgold) [inline google/cloud/forseti/notifier/notifiers/cscc_notifier.py]: ditto on the `if`

File: google/cloud/forseti/notifier/notifiers/cscc_notifier.py [hunk 5]
```diff
@@ -258,14 +209,3 @@ def run(self, violations, gcs_path=None, mode=None, organization_id=None,
             LOGGER.debug('Running CSCC with beta API. source_id: %s', source_id)
             self._send_findings_to_cscc(violations, source_id=source_id)
             return
-
-        # alpha api
```
Comment (blueandgold) [inline google/cloud/forseti/notifier/notifiers/cscc_notifier.py]: ditto on the `if`

File: configs/server/forseti_conf_server.yaml.in [hunk 6]
```diff
@@ -523,15 +523,10 @@ notifier:
       cscc:
         enabled: false
         mode: api
-        # Alpha API
```
Comment (blueandgold) [inline configs/server/forseti_conf_server.yaml.in]: Can you also please delete the alpha discover doc, as below?  There is no version number.
https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/google/cloud/forseti/common/gcp_api/discovery_documents/securitycenter.json

**Comment by red2k18** [comment]
I think the `if` condition to check if Beta API is being used or not, will be helpful for testing when we move to GA (very soon) and will make Beta deprecation easier.

**Comment by red2k18** [comment]
Addressed all the comments in #2647

**Comment by red2k18** [comment]
Closing this PR. Opened #2647 instead.

**Pull request closed without merging**